packages/pds-ember/addon/components/pds/global-footer/index.js

```diff
@@ -0,0 +1,6 @@
+import Component from '@glimmer/component'
+
+/**
+ * @class PdsGlobalFooter
+ */
+export default class PdsGlobalFooter extends Component {}
```

Comment:
When a component class is empty this you don't technically need to have the file. Is there a reason we add these in structure and cloud-ui @meirish?

Reply:
They aren't absolutely necessary, but there have been strange quirks with Storybook integration when they're not present (specifically with the YUIDoc comments).  Additionally, it just removes an extra step later on if/when we need to add behavior to a component.